.github/CODEOWNERS

```diff
@@ -367,6 +368,9 @@
 /isdown/*metadata.csv                    @ntomas support@isdown.app @DataDog/documentation @DataDog/ecosystems-review
 /isdown/manifest.json                    @ntomas support@isdown.app @DataDog/documentation @DataDog/ecosystems-review
 /isdown/README.md                        @ntomas support@isdown.app @DataDog/documentation @DataDog/ecosystems-review
+/jfrog_platform_cloud/*metadata.csv      @jfrog/partner-engineering @DataDog/documentation
+/jfrog_platform_cloud/manifest.json      @jfrog/partner-engineering @DataDog/documentation
+/jfrog_platform_cloud/README.md          @jfrog/partner-engineering @DataDog/documentation
```

Comment:
please add @DataDog/ecosystems-review to the codeowners. Thanks!

Reply:
Added for ll JFrog records in CODEOWNERS.